content/en/docs/ops/integrations/certmanager/index.md

```diff
@@ -3,6 +3,9 @@ title: cert-manager
 description: Information on how to integrate with cert-manager.
 weight: 20
 keywords: [integration,cert-manager]
+aliases:
+  - /docs/tasks/traffic-management/ingress/ingress-certmgr
```

Comment:
this task should terminate with `/` (to fix linting, as well as require less lookup on redirects.

Cheers
-steve